tests/test_https.py

```diff
@@ -75,7 +75,7 @@ def test_not_trusted_ca():
     not hasattr(tests.ssl_context(), "minimum_version"),
     reason="ssl doesn't support TLS min/max",
 )
-def test_set_min_tls_version():
+def test_set_min_tls_version_str():
```

Comment:
Instead of repeating code for enum you could do this.

```suggestion
@pytest.mark.parametrize("version", (None, ssl.TLSVersion.TLSv1_2, "TLSv1_2"))
def test_set_min_tls_version(version):
```

```diff
...
-        http = httplib2.Http(tls_minimum_version="TLSv1_2")
+        http = httplib2.Http(tls_minimum_version=version)
```

Reply:
My pytest skills are rusty. Thank you for helping me with being more idiomatic @temoto. It is appreciated. I'll try to keep this in mind for any future submissions.